examples/README.md

```diff
@@ -248,6 +249,70 @@ For this example the following is needed in the selected region:
 * Compute Engine API: Resource policies: **one for each job in parallel** -
   _only needed for `compute` partition_
 
+### [slurm-gcp-v5-high-io.yaml] ![community-badge]
+
+This example uses [Slurm on GCP][slurm-gcp] version 5.x modules to replicate the
+[hpc-cluster-high-io.yaml] core example. With version 5, additional features are
+available and utilized in this example:
+
+* node groups are used to allow multiple machine types in a single partition,
+  differentiated by node names.
+* Active cluster reconfiguration is on by default. When updating a partition or
+  cluster configuration, the overwrite option (`-w`) can be used and upon
+  re-applying the deployment, the changes will become active without having to
+  destroy and recreate the cluster.
+
+This blueprint will create a cluster with the following storage tiers:
+
+* The homefs mounted at `/home` is a default "BASIC_HDD" tier filestore with
+  1 TiB of capacity
+* The projectsfs is mounted at `/projects` and is a high scale SSD filestore
+  instance with 10TiB of capacity.
+* The scratchfs is mounted at `/scratch` and is a
+  [DDN Exascaler Lustre](../community/modules/file-system/DDN-EXAScaler/README.md)
+  file system designed for high IO performance. The capacity is ~10TiB.
+
+The cluster will support 2 partitions:
+
+* `lowcost`
```

Comment:
Add documentation to node group documentation on motivation (e.g. heterogeneous jobs - head node, worker nodes), add pointer to it here.